docs/docs/modeling.md

```diff
@@ -118,4 +118,28 @@ classDiagram
 
 ## AssignedCategory -　
 
-## AssignedTask -
+## AssignedTask - 割当タスク
+
+### taskId
+
+- TaskId is the UUID
+
+### title
+
+- The owner can set a title as they prefer. This title can be modified at any time.
+
+Constraints:
+
+- It must be between 1 and 20 characters in length.
+
+### description
+
+- The owner can provide details for the task. This description can be modified at any time.
+
+Constraints:
+
+- The description must be between 10 and 120 characters.
```

Comment:
I'm not sure whether we should write about `taskId`, `title` and `description` here too as they are exactly the same as those in `Task`.
It might be better to mention that these descriptions and constraints should be referred to in `Task`.

Reply:
Thanks for reviewing. I completely forgot about it.